src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -179,10 +181,26 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeIsReadOnlyAttribute(this));
             }
 
+            // Synthesize `ParamArrayAttribute`.
```

Comment:
Thanks. I would say it's still much faster to read that one sentence than parse the whole block of code to see what it's doing. But I guess here it's obvious since it's what the rest of the method is doing. So, I removed the comments.